modules/Hoot/ui/managePanel/TaskingManagerPanel.js

```diff
@@ -0,0 +1,373 @@
+import Tab from './tab';
+import { geoExtent as GeoExtent } from '../../../geo';
+import { d3combobox } from '../../../lib/hoot/d3.combobox';
+
+/**
+ * Creates the tasking manager tab in the settings panel
+ *
+ * @extends Tab
+ * @constructor
+ */
+export default class TaskingManagerPanel extends Tab {
+    constructor( instance ) {
+        super( instance );
+
+        this.name = 'Tasking Manager';
+        this.id   = 'taskingManagerPanel';
+        this.currentProject = null;
+
+        this.deriveTypeOpts = {
+            name: 'deriveType',
+            readonly: 'readonly',
+            options: [ 'Adds only', 'Cut & Replace', 'Differential changeset' ]
```

Comment:
Let's remove "changeset" from the last option.  Thinking on these terms "Adds only" might be confused with Differential.  Might need to think about these more.  Or have tool tips to show details of what they mean.